fireapp/lib/presentation/unavailability_form/unavailability_form_widget.dart

```diff
@@ -67,125 +48,142 @@ class _UnavailabilityFormState extends FireAppState<UnavailabilityForm>
+                            decoration:
+                                textFieldStylePositioned(context).copyWith(
+                              prefixIcon: const Icon(Icons.calendar_today),
+                              labelText: AppLocalizations.of(context)
+                                  ?.enterUnavailabilityStartDate,
+                            )
+                        );
+                      }),
+                  StreamFormField<TimeOfDay?>(
+                      stream: viewModel.selectedStartTime,
+                      builder: (context, value) {
+                        return TimeFormField(
+                          currentValue: value,
+                          onValueChanged: viewModel.updateStartTime,
                           decoration:
                               textFieldStylePositioned(context).copyWith(
-                            prefixIcon: const Icon(Icons.calendar_today),
+                            prefixIcon: const Icon(Icons.hourglass_top),
                             labelText: AppLocalizations.of(context)
-                                ?.enterUnavailabilityStartDate,
-                          )
-                      );
-                    }),
-                StreamFormField<TimeOfDay?>(
-                    stream: viewModel.selectedStartTime,
-                    builder: (context, value) {
-                      return TimeFormField(
-                        currentValue: value,
-                        onValueChanged: viewModel.updateStartTime,
-                        decoration: textFieldStylePositioned(context).copyWith(
-                          prefixIcon: const Icon(Icons.hourglass_top),
-                          labelText: AppLocalizations.of(context)
-                              ?.enterUnavailabilityStartTime,
-                        ),
-                      );
-                    }),
-                StreamFormField<DateTime?>(
-                    stream: viewModel.selectedEndDate,
-                    builder: (context, value) {
-                      return DateFormField(
+                                ?.enterUnavailabilityStartTime,
+                          ),
+                        );
+                      }),
+                  StreamFormField<DateTime?>(
+                      stream: viewModel.selectedEndDate,
+                      builder: (context, value) {
+                        return DateFormField(
+                            currentValue: value,
+                            onValueChanged: viewModel.updateEndDate,
+                            decoration:
+                                textFieldStylePositioned(context).copyWith(
+                              prefixIcon: const Icon(Icons.calendar_today),
+                              labelText: AppLocalizations.of(context)
+                                  ?.enterUnavailabilityEndDate,
+                            )
+                        );
+                      }),
```

Comment:
Same thing with brackets

Reply:
I'm having problems with dart format and my IDE making it impossible to indent in the way you're asking, not sure how to go around this.